zhaquirks/__init__.py

```diff
@@ -264,7 +267,7 @@ def handle_cluster_request(
         ] = None,
     ):
         """Handle the cluster command."""
-        if hdr.command_id == ZONE_STATE:
+        if hdr.command_id == ZONE_STATUS_CHANGE_COMMAND:
             if self._timer_handle:
                 self._timer_handle.cancel()
             self._timer_handle = self._loop.call_later(self.reset_s, self._turn_off)
```

Comment:
That must only be performed if the `zone_status` is one of `Alarm_1` or `Alarm_2`. For any other values, the alarm status must be reset. 
(I'm thinking in a `Tamper` or `Battery` status)

Reply:
Not sure I've ever had a device that actually uses the other ones, but you're right. See https://github.com/zigpy/zha-device-handlers/pull/2231/commits/7ca79f64b6fa07010849d2bfb4cc3bdc7d92cf44